### PR TITLE
Topic/el6 experiment

### PR DESCRIPTION
**Pull request by danzelenak-usgs**
Just so we can compare CentOS 6 builds, figure out which one we want to go with.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
-# Install some remaining dependencies
-RUN yum -y install netcdf4-python && \
-    yum -y install python2-numpy && \
-    yum -y install scipy && \
-    pip install scipy
+# We need to clear out this older version of numpy
+RUN rm -rf /usr/local/lib/python2.7/site-packages/numpy*
+RUN /usr/local/bin/pip install numpy==1.16.2 lxml==3.6.0 netcdf4==1.4.2
+
+RUN yum -y install modtran-espa --nogpgcheck
```
Comment (caustin-usgs) [inline Dockerfile]: should the modtran-espa install be included in the playbook task?

File: playbook/espa-worker/tasks/espa-software.yml [hunk 2]
```diff
@@ -115,12 +133,46 @@
     name: "espa-plotting-{{ espa_plotting }}"
     state: installed
 
-# Install modtran
```
Comment (caustin-usgs) [inline playbook/espa-worker/tasks/espa-software.yml]: here's where it went away.... i don't think we need openjdk either now that were not running on hadoop

File: Dockerfile (hunk 1 above)
Comment (danzelenak-usgs) [reply Dockerfile]: re-added modtran installation to playbook

File: playbook/espa-worker/tasks/espa-software.yml (hunk 2 above)
Comment (danzelenak-usgs) [reply playbook/espa-worker/tasks/espa-software.yml]: Commented out jdk installation

File: Dockerfile (hunk 1 above)
Comment (caustin-usgs) [inline Dockerfile]: can this RUN cmd go away?

Comment (danzelenak-usgs) [reply Dockerfile]: Removed this line

**Pull request merged**